web-transport-quinn/src/client.rs

```diff
@@ -21,6 +24,7 @@ pub enum CongestionControl {
 ///
 /// This is optional; advanced users may use [Client::new] directly.
 pub struct ClientBuilder {
```

Comment:
Can you gate `ClientBuilder` on aws-lc-rs or ring instead of individual fields/functions? It's just impossible to use without those features, right?